docsets/LiveCode/docset.json

```diff
@@ -9,25 +9,21 @@
  
     "specific_versions": [ 
     
+             {
+             "version": "1.9.1",
+             "archive": "versions/1.9.1/LiveCode.tgz"
```

Comment:
This archive doesn't exist. I think you added `versions/1.8.1/LiveCode.tgz` instead.

Reply:
Darn. No I will se what I did wrong.
I will hopefully resolve this tomorrow When I look at what I did, and still have.